docs/content/docs/osm_config_map.md

```diff
@@ -4,24 +4,154 @@ description: "OSM ConfigMap"
 type: docs
 ---
 
-# OSM ConfigMap
 
-OSM deploys a configMap `osm-config` as a part of its control plane (in the same namespace as that of the osm-controller pod) which can be updated by the mesh owner/operator at any time. The purpose of this configMap is to provide the mesh owner/operator the ability to update some of the mesh configurations based on their needs.
+Table of Contents
```

Comment:
It seems like our website automatically adds a ToC to the right hand side of the page so it may be cleaner to remove this?
https://deploy-preview-2940--osm-docs.netlify.app/docs/osm_config_map/

Reply:
oops didn't realize, will go ahead and remove it